### DISCUSSION TITLE
Quantification plexDIA SILAC DIA-NN 1.9

### DISCUSSION THREAD

**Discussion by alm3240** [vdemichev/DiaNN#1208] [General]
After searching plexDIA runs with light/heavy SILAC, in silico library build with no modification and additional notes: 
--fixed-mod SILAC,0.0,KR,label
--lib-fixed-mod SILAC
--channels SILAC,L,KR,0:0; SILAC,H,KR,8.014199:10.008269
--peak-translation
--original-mods
--channel-spec-norm

What are the values in report.tsv called "Channel.L" "Channel.H"? For protein summarization with iq::process_long_format, should I split light/heavy precursor first and use intensity_col="Precursor.Normalised" or is there a better column for quantification? Should I  filter on Global.Q.Value, Global.PG.Q.Value and Channel.Q.Value?

**Comment by vdemichev** [comment]
Hi,

Please ignore those and instead use Precursor.Normalised in the main .parquet report. Those values are, I think, extracted MS1 signal at the indicated RT for the respective channel, but I am not sure, it's some old code that generates those and it has not been validated for more than a year.

> For protein summarization with iq::process_long_format

You can use directly PG.MaxLFQ from the .parquet report, so long as DIA-NN processing was performed with QuantUMS enabled.

> --channel-spec-norm

So SILAC channels correspond to two different conditions?

> Should I filter on Global.Q.Value, Global.PG.Q.Value and Channel.Q.Value?

Please see https://github.com/vdemichev/DiaNN?tab=readme-ov-file#match-between-runs

Best,
Vadim
